### ISSUE TITLE
CSS3 support

### ISSUE THREAD

**Issue by silverki** [alanshaw/markdown-pdf#59]
Have I understood correctly that there is no CSS3 support?

**Comment by silverki** [comment]
Yes there is. Issue is that @font-face font is not used in header and footer.

**Comment by anko** [comment]
Could you create a simple test case?

**Comment by timactive** [comment]
+1

**Comment by PradyumnaR** [comment]
Unable to insert image tag in header and footer.....
#markdown options
options{
runningsPath: path.join('runnings.js')
}
#runnings.js
exports.header = {
content: function (..,..){
return "<image>"
}
}

**Comment by maethor** [comment]
Hi,

You said it's possible, but how do you apply css on header and footer? What is the class or the id?

Thanks.

**Comment by PradyumnaR** [comment]
Hi maethor,
I was able to solve this issue. Header and footer supports only inline css styling we cannot import or do external styling.
check this link for more detailed work on header and footer:
https://jsreport.net/learn/phantom-pdf

**Comment by maethor** [comment]
Thank you @PradyumnaR 

Indeed, I also managed it with inline css.

This seems strange, because https://atom.io/packages/markdown-themeable-pdf, which seems to works just like this, manage css in footer pretty nicely.

**Comment by neooblaster** [comment]
Hi All,

I'm also interested for CSS support for headers/footers to centralize the customisation, using ID/Class/Tag.

Thanks for all !
